source/sink-connector/fundamentals/write-strategies.txt

```diff
@@ -35,11 +35,40 @@ To learn how to modify the sink records your connector receives before your
 connector writes them to MongoDB, read the guide on
 :ref:`Sink Connector Post Processors <sink-fundamentals-post-processors>`.
 
-To learn more about bulk writes, see the MongoDB Java driver guide on
-`Bulk Writes <{+connector_driver_url_base+}fundamentals/crud/write-operations/bulk/>`__.
-
 To see a write model strategy implementation, see the source code of the
-:github:`InsertOneDefaultStrategy class <mongodb/mongo-kafka/blob/master/src/main/java/com/mongodb/kafka/connect/sink/writemodel/strategy/InsertOneDefaultStrategy.java>`.
+:github:`InsertOneDefaultStrategy class
+<mongodb/mongo-kafka/blob/master/src/main/java/com/mongodb/kafka/connect/sink/writemodel/strategy/InsertOneDefaultStrategy.java>`.
+
+.. _sink-connector-bulk-write-ops:
+
+Bulk Write Operations
+---------------------
+
+The sink connector writes data to MongoDB using bulk write operations.
+Bulk writes group multiple write operations, such as inserts,
+updates, or deletes, together. 
+
+By default, the sink connector performs ordered bulk writes, which
+guarantees the order of data changes. In an ordered bulk write, if any write operations
+results in an error, the connector skips the remaining writes in that
+batch. Because of this behavior, there are performance benefits to
+using unordered bulk writes, if you can make changes to your data in any
+order. The sink connector can perform unordered bulk writes in parallel, which can increase
+performance. In addition, if you set the ``errors.tolerance`` setting to
+``all`` and any write operation in your bulk write fails, the connector continues to
+perform the remaining write operations in the batch that do not return errors.
+
+If you don't need to guarantee the order of data changes, you can
+specify the ``bulk.write.ordered`` setting to perform unordered bulk
+writes instead. To learn more about this setting, see the
+:ref:`Connector Message Processing Properties
+<sink-configuration-message-processing>`.
```

Comment:
I started trying to explain my suggestions here but thought it would be easier if I just showed you how I would do it (which doesn't necessarily make it "right" but makes it clearer to me personally, and maybe others too). Feel free to use as written or change. Questions are inline.

```suggestion
By default, the sink connector performs ordered bulk writes, which
guarantee the order of data changes. In an ordered bulk write, if any write operation
results in an error, the connector skips the remaining writes in that
batch. 

If you don't need to guarantee the order of data changes, you can
[set?] the ``bulk.write.ordered`` setting [to false?] to perform unordered bulk
writes instead. The sink connector can [can? or does it always do so?] perform unordered bulk writes in parallel, which can improve performance. 

In addition, if you set the ``errors.tolerance`` setting to
``all`` and any write operation in your bulk write fails, the connector continues to
perform the remaining write operations in the batch that do not return errors. [Is this true of both ordered and unordered bulk writes?]

.. tip:: To learn more about the ``bulk.write.ordered`` setting, see the
:ref:`Connector Message Processing Properties
<sink-configuration-message-processing>`.
```